invenio_vocabularies/contrib/subjects/datastreams.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021-2022 CERN.
+# Copyright (C) 2024 University of Münster.
+#
+# Invenio-Vocabularies is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Names datastreams, transformers, writers and readers."""
+
+import idutils
+from invenio_access.permissions import system_identity
+from invenio_i18n import lazy_gettext as _
+
+from ...datastreams.errors import TransformerError
+from ...datastreams.transformers import BaseTransformer
+from ...datastreams.writers import ServiceWriter
+from .config import subject_schemes
+
+
+class YAMLTransformer(BaseTransformer):
```

Comment:
Also: YAML is ... actually kind of terrible as an data interchange format. Switching to CSV for my the subjects I manage has cut load time by 99% (literally not kidding) and space by around 66%. I perfectly understand this is what datastreams deal with originally, so it's not "a choice" in this work initially and I don't expect adjustments for that. Just sharing is caring. (this is also in part why I never bothered with datastreams...)

Reply:
Yes, I see. We will replace this locally with a Marc21Transformer. It's just some sort of "base implementation", a change is left to another PR.